setup.py

```diff
@@ -86,7 +86,6 @@ def get_package_data(package):
         "Operating System :: OS Independent",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
Add 3.10?

Reply:
that is already added see a bit further below.

Reply:
Got caught by the context diff...